contracts/ITablelandController.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+/**
+ * @dev Interface of a TablelandController compliant contract.
+ *
+ * This interface can be implemented to enabled advanced access control for a table.
+ * Call {ITablelandTables-setController} with the address of your implementation.
+ *
+ * See {test/TestTablelandController} for an example of token-gating table write-access.
+ */
+interface ITablelandController {
```

Comment:
moved this to an interface instead of an abstract contract. this means that if the proxy is upgraded, controller contracts that refer to this interface will continue to serve policies to the tables contract. previously, this was not the case since the controller contract would refer to a contract referenced by the previous table implementation 😅